### PR TITLE
fine-toothed comb pass, including addressing comments in #112

### PR DESCRIPTION
**Pull request by petrelharp**
I'm going through the whole thing - currently up to the Methods but will continue. Looks like we've got a conflicting PR in #126.

### CODE REVIEW

**Comment by petrelharp** [comment]
Done going through; will reconcile with #126 now.

**Comment by petrelharp** [comment]
I've rebased against #126; so this should apply cleanly after merging Adam's edits.

**Comment by andrewkern** [comment]
reviewing this now

File: popSimManu.tex
Comment (andrewkern) [inline popSimManu.tex:140]: should we add "Standardization" or "Standards"?

File: popSimManu.tex
Comment (andrewkern) [inline popSimManu.tex]: unclear to what "this" refers back to in this context. I'd keep it "this approach"

File: popSimManu.tex
Comment (andrewkern) [inline popSimManu.tex:267]: maybe add a quick sentence here that says "Adding further maps into the resource is trivial"

File: popSimManu.tex
Comment (andrewkern) [inline popSimManu.tex:289]: what if we make this a positive thing and say "to allow simulation of processes that can only be modeled forward in time"?

File: popSimManu.tex
Comment (andrewkern) [inline popSimManu.tex]: should we give the readers a link to the documentation on this?

File: popSimManu.tex
Comment (andrewkern) [inline popSimManu.tex:426]: see main text? see methods?

File: popSimManu.tex
Comment (andrewkern) [inline popSimManu.tex]: D. melanogaster for consistency

**Review by andrewkern — APPROVED** [review]
some small comments in here but one bigger thing we are missing: we don't have a link to the `stdpopsim` repo! I can add that in a separate PR if you are busy @petrelharp

File: popSimManu.tex
Comment (agladstein) [inline popSimManu.tex]: I would make this active (since it's not automatic). (e.g "Developer A submits a quality control issue...")

**Review by agladstein — APPROVED** [review]
Lots of good edits here. I think it reads much better.

File: popSimManu.tex
Comment (petrelharp) [reply popSimManu.tex:140]: is that a keyword? dunno.

File: popSimManu.tex
Comment (petrelharp) [reply popSimManu.tex]: it's not an "approach"; that's the problem. Changing to "this situation".

File: popSimManu.tex
Comment (petrelharp) [reply popSimManu.tex:289]: well, that's not quite the same thing? I vote to keep this.

File: popSimManu.tex
Comment (petrelharp) [reply popSimManu.tex]: a bit unwieldy, but ok: https://stdpopsim.readthedocs.io/en/latest/development.html#naming-conventions

File: popSimManu.tex
Comment (petrelharp) [reply popSimManu.tex:426]: I think this is OK.

**Comment by petrelharp** [comment]
I've addressed your comments above? I can squash if you like.

File: popSimManu.tex
Comment (agladstein) [inline popSimManu.tex]: This hyper ref is not displaying the link.
Screen Shot 2019-12-20 at 1 15 17 PM

Comment (petrelharp) [reply popSimManu.tex]: whoops; meant to do `url`

**Review by agladstein — APPROVED** [review]

**Comment by petrelharp** [comment]
Here's a squash. Should be ready to merge after #126.

**Pull request merged**
